jopa-maven-plugin/pom.xml

```diff
@@ -30,6 +30,22 @@
             <artifactId>jopa-owl2java</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-project</artifactId>
+            <version>2.2.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-compat</artifactId>
+            <version>3.2.5</version>
+        </dependency>
```

Comment:
What are these two new dependencies for? Are they necessary?

Reply:
MavenProject is for using MavenProject object in plugin, which is used for getting TestClasspath and and compile source roots

Reply:
Maven compat is for DependencyResolutionRequiredException which can be thrown by getting TestClasspath